text/0000-pack-buildpack-create.md

```diff
@@ -0,0 +1,80 @@
+# Meta
+[meta]: #meta
+- Name: Pack Command to Create a Buildpack Repo
+- Start Date: 2021-01-19
+- Author(s): [jkutner](https://github.com/jkutner)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a proposal for a new command in Pack that would create the scaffolding for a new buildpack.
+
+# Definitions
+[definitions]: #definitions
+
+- *Scaffold* - the essential files and directory required for a buildpack
+
+# Motivation
+[motivation]: #motivation
+
+Every new buildpack requires a few essential files and directories. We can make buildpack development easier by automatically generating these with a single command.
+
+# What it is
+[what-it-is]: #what-it-is
+
+Add a new command to Pack:
+
+```
+$ pack buildpack create <id> \
+    --language <lang> \
+    --stack <stack>
+```
+
+Where the flags include:
+
+* `language` - a choice of programming languages to use for the template. Pack will support Bash and Go at a minimum.
+* `stack` - a set of compatible stacks for the buildpack. may be specificied multiple times.
+* `path` - the location on the filesystem to generate the artifacts
```

Comment:
The default path would be the current directory?

Reply:
I assume so, but it'd be good to specify.

```
$ pack buildpack create <foo>
```
would make a directory `foo/` in the current working dir.

Reply:
@hone agree, I wasn't sure how to handle this because the id `foo/bar` shouldn't create two directories (`./foo/bar/`), and I thought `foo_bar` was kind of ugly. I noticed that the Paketo Bootstrapper would create a one directory: `bar/`. I like that

Reply:
good call, forgot about the namespace bit.